lib/scripts/update_datadog_config.rb

```diff
@@ -0,0 +1,60 @@
+# Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2.0 License.
+# This product includes software developed at Datadog (https://www.datadoghq.com/).
+# Copyright 2022-Present Datadog, Inc.
+
+#!/usr/bin/env ruby
+
+require 'yaml'
+
+NODE_AGENT_TAGS_FILE = "/home/vcap/app/.datadog/node_agent_tags.txt"
+DD_TAGS_FILE = "/home/vcap/app/.datadog/.dd_tags.txt"
+
+def sanitize(tags_env_var, delimiter)
+  tags_list = tags_env_var.gsub(",\"", ";\"").split(delimiter)
+  tags_list.keep_if { |element| !element.include?(";") }
+  tags_list = tags_list.map { |tag| tag.gsub(" ", "_") }
+  return tags_list.uniq
+end
+
+def read_yaml_file(file_path)
+  yaml_file = File.read(file_path)
+  YAML.load(yaml_file)
+end
+
+def write_yaml_file(file_path, data)
+  File.write(file_path, YAML.dump(data))
+end
+
+def get_tags()
+  dd_tags = File.file?(DD_TAGS_FILE) ? File.read(DD_TAGS_FILE) : nil
+  dd_node_agent_tags = ENV['DD_NODE_AGENT_TAGS'] || (File.file?(NODE_AGENT_TAGS_FILE) ? File.read(NODE_AGENT_TAGS_FILE) : nil)
+
+  tags = []
+
+  if !dd_tags.nil?
+    tags += sanitize(dd_tags, ",")
```

Comment:
This delimiter is not guaranteed to be a comma as it is coming from the user input

Reply:
It actually is, the `.dd_tags.txt` file contains the output of the `get_tags.py` script (always comma-separated)


```bash
# lib/run-datadog.sh
export DD_TAGS=$(python "${DATADOG_DIR}/scripts/get_tags.py")
echo "${DD_TAGS}" > "${DATADOG_DIR}/.dd_tags.txt"
```

Reply:
Oh nice, that's good!